setup.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from setuptools import setup
+
+from qiime2 import __version__ as qiime2_version
```

Comment:
I can do that just note that we are doing this based on your suggestion ... pr in a second ...